great_expectations/rule_based_profiler/types/data_assistant_result.py

```diff
@@ -23,6 +26,38 @@ class DataAssistantResult(SerializableDictDot):
     ] = None  # Obtain "meta/details" using "meta = expectation_suite.meta".
     execution_time: Optional[float] = None  # Execution time (in seconds).
 
+    def plot(self, prescriptive: bool = False):
+        metrics: Dict[Dict, ParameterNode] = self.metrics
+        metric_names: List[str] = []
+        metric_domain: Domain
+        metric_nodes: Dict[str, ParameterNode]
+        parameter_node_name: str
+        parameter_node: ParameterNode
+        for metric_domain, metric_nodes in metrics.items():
+            for parameter_node_name, parameter_node in metric_nodes.items():
+                if parameter_node_name.endswith(".details"):
+                    details_parameter_node: ParameterNode = parameter_node
+                    metric_names.append(
+                        details_parameter_node.metric_configuration.metric_name
+                    )
+                if parameter_node_name.endswith(".attributed_value"):
```

Comment:
@NathanFarmer I think that the metric name can be parsed using a regular expression and the constants, such as `FULLY_QUALIFIED_PARAMETER_NAME_DELIMITER_CHARACTER`, `FULLY_QUALIFIED_PARAMETER_NAME_SEPARATOR_CHARACTER`, `PARAMETER_NAME_ROOT_FOR_PARAMETERS`, `FULLY_QUALIFIED_PARAMETER_NAME_ATTRIBUTED_VALUE_KEY`, `FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY`, etc.  Happy to discuss.  Thanks.

Reply:
✅